src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/migration/Cas3UpdateApplicationOffenderNameJob.kt

```diff
@@ -0,0 +1,84 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.migration
+
+import org.apache.commons.collections4.ListUtils
+import org.slf4j.LoggerFactory
+import org.springframework.data.domain.PageRequest
+import org.springframework.data.domain.Slice
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.ApplicationRepository
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.TemporaryAccommodationApplicationEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.PersonSummaryInfoResult
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.problem.ForbiddenProblem
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.problem.NotFoundProblem
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.OffenderService
+import java.util.stream.Collectors
+import javax.persistence.EntityManager
+
+class Cas3UpdateApplicationOffenderNameJob(
+  private val applicationRepository: ApplicationRepository,
+  private val offenderService: OffenderService,
+  private val entityManager: EntityManager,
+  private val pageSize: Int,
+) : MigrationJob() {
+  private val log = LoggerFactory.getLogger(this::class.java)
+  override val shouldRunInTransaction = false
+
+  @SuppressWarnings("MagicNumber", "TooGenericExceptionCaught")
+  override fun process() {
+    var page = 1
```

Comment:
would be useful to log `start` of migration and `completion` of migration, so that we knew the process is completed.

Reply:
done